README

```diff
@@ -34,9 +34,14 @@ II. Misc
 
 The layer contains three recipes for OpenDDS: two versioned, one not.
 
-To select the versioned recipes for OpenDDS 3.22, 3.20 or 3.14, include
+To select the versioned recipes for OpenDDS 3.24, 3.22, 3.20 or 3.14, include
```

Comment:
Is there a need to keep old OpenDDS versions?

Reply:
I will remove the other 3.14, 3.20 and 3.22 scripts